dna-frontend/src/app/core/components/appbar/appbar.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+import { TuiAppBarModule } from '@taiga-ui/experimental';
+import { TuiSvgModule } from '@taiga-ui/core';
+
+@Component({
+  selector: 'app-appbar',
+  standalone: true,
+  imports: [TuiAppBarModule, TuiSvgModule],
+  templateUrl: './appbar.component.html',
+  styleUrl: './appbar.component.scss',
+})
+export class AppbarComponent {
+  @Input() pageName = "It's an app bar";
```

Comment:
Probably change these templated strings to empty strings now that we are merging these components into main